srcpkgs/picard/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'picard'
 pkgname=picard
-version=2.9.1
-revision=2
+version=2.10.0rc1
```

Comment:
The `2.10` version has been released today. The newer version should be preferred I believe.

Reply:
I updated it accordingly.